recipes/fast_double_parser/all/conanfile.py

```diff
@@ -1,5 +1,8 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
 import os
-from conans import ConanFile, tools
 
 required_conan_version = ">=1.43.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
check_min_cppstd: https://github.com/conan-io/conan-center-index/blob/master/docs/v2_linter.md